src/site/content/en/blog/contact-picker/index.md

```diff
@@ -0,0 +1,316 @@
+    <source type="video/webm"
+      src="https://storage.googleapis.com/webfundamentals-assets/contact-picker/contact-picker.webm">
+    <source type="video/mp4"
+      src="https://storage.googleapis.com/webfundamentals-assets/contact-picker/contact-picker.mp4">
+  </video>
+</a>
+
+Access to the user's contacts has been a feature of native apps since
+(almost) the dawn of time. It's one of the most common feature requests
+I hear from web developers, and is often the key reason they build a native
+app.
+
+The Contact Picker API is an on-demand API that allows users to
+select entries from their contact list and share limited details of the
+selected entries with a website. It allows users to share only what they
+want, when they want, and makes it easier for users to reach and connect
+with their friends and family.
+
+For example, a web-based email client could use the Contact Picker API to
+select the recipient(s) of an email. A voice-over-IP app could look up
+which phone number to call. Or a social network could help a user discover
+which friends have already joined.
+
+{% Aside 'caution' %}
+  We've put a lot of thought into the design and implementation of the
+  Contact Picker API to ensure that the browser will only share exactly
+  what people choose. See the [Security and Privacy](#security-considerations)
+  section below.
+{% endAside %}
+
+## Current status {: #status }
+
+| Step                                       | Status                       |
+| ------------------------------------------ | ---------------------------- |
+| 1. Create explainer                        | [Complete][explainer]        |
+| 2. Create initial draft of specification   | [In Progress][spec]          |
+| 3. Gather feedback & iterate on design     | [In progress][spec]          |
+| **4. Origin trial**                        | **Started in Chrome 77** <br> Expected to run through Chrome 80. |
+| 5. Launch                                  | Not started                  |
+
+## Using the Contact Picker API {: #how-to-use }
+
+The Contact Picker API requires a single API call with an options parameter
+that specifies the types of contact information you want.
+
+{% Aside 'codelab' %}
+  Check out the [Contact Picker API demo](https://contact-picker.glitch.me)
+  and view the
+  [source](https://glitch.com/edit/#!/contact-picker?path=demo.js:20:0).
+{% endAside %}
+
+### Enabling via chrome://flags
+
+To experiment with the Contact Picker API locally, without an origin
+trial token, enable the `#enable-experimental-web-platform-features` flag
+in `chrome://flags`.
+
+### Enabling support during the origin trial phase {: #origin-trial }
+
+Starting in Chrome 77, the Contact Picker API is available as an origin
+trial on Chrome for Android. Origin trials allow you to try new features
+and give feedback on their usability, practicality, and effectiveness, both
+to us, and to the web standards community. For more information, see the
```

Comment:
```suggestion
to the Chrome team, and to the web standards community. For more information, see the
```

Avoiding "we" per Rob's rationale in https://github.com/GoogleChrome/web.dev/pull/1614/files#r333143069